### PR TITLE
fix: [PIE-8738]: flaky jest tests skip

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

> Test DetailsPanel :- 70 
> <ExecutionStatusLabel /> tests :- 34 
> ExecutionStatusIcon :- 34 
> HealthCheck table tests :- 27 
> FlagConfigurationStepWidget :- 24 

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/AmazonS3Artifact/__tests__/AmazonS3.test.tsx [hunk 1]
```diff
@@ -93,7 +93,8 @@ const doConfigureOptionsTesting = async (cogModal: HTMLElement, fieldElement: HT
   await waitFor(() => expect(fieldElement.value).toBe('<+input>.regex(<+input>.includes(/test/))'))
 }
 
-describe('AmazonS3 tests', () => {
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('AmazonS3 tests', () => {
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/AmazonS3Artifact/__tests__/AmazonS3.test.tsx:97]: Why are we skipping all the test cases ? Can't we skip the one that is failing?

Comment (vikrant-harness) [reply src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/AmazonS3Artifact/__tests__/AmazonS3.test.tsx:97]: We extracted the title information of the tests as shown on the jest dashboard , Let me check if we can pinpoint to the exact test

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by PratyushGar — DISMISSED** [review]

**Review by jyotiarora-harness — APPROVED** [review]

**Review by knagurski — CHANGES_REQUESTED** [review]
We definitely do not want to kill off entire suites for a single flaky tests.

File: src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/__tests__/FlagConfigurationStepWidget.test.tsx [hunk 2]
```diff
@@ -99,7 +99,8 @@ describe('FlagConfigurationStepWidget', () => {
     expect(screen.getByTestId('flag-configuration-step-widget-error')).toBeVisible()
   })
 
-  test('it should render the form', async () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  test.skip('it should render the form', async () => {
```
Comment (knagurski) [inline src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/__tests__/FlagConfigurationStepWidget.test.tsx:103]: Looks like this one is below the 20 threshold. Can we re-enable?

Comment (vikrant-harness) [reply src/modules/75-cf/components/PipelineSteps/FlagConfigurationStep/__tests__/FlagConfigurationStepWidget.test.tsx]: Sorry , My bad enabled it.

**Review by knagurski — APPROVED** [review]
Thanks for doing this. Flaky tests are the worst!

**Review by PratyushGar — APPROVED** [review]

**Pull request merged**